### PR TITLE
Create 2024-06-20-performance-portability-training-series.md

### PR DESCRIPTION
**Pull request by codeplaymax**
This PR adds a blog for the Performance and Portability Training Series.

### CODE REVIEW

File: content/news/2024/2024-06-20-performance-portability-training-series.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+contributor: max
+date: '2024-06-20T14:10:22.153253'
```
Comment (scottstraughan) [inline content/news/2024/2024-06-20-performance-portability-training-series.md:3]: Is this the correct date?

File: content/news/2024/2024-06-20-performance-portability-training-series.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+external_url: https://www.alcf.anl.gov/events/portable-sycl-code-using-onemkl-amd-intel-and-nvidia-gpus
+title: 'Portable SYCL Code Using oneMKL on AMD, Intel, and Nvidia GPUs'
+image: ../../../static/images/news/2024-06-20 performance-portability-series.webp
```
Comment (scottstraughan) [inline content/news/2024/2024-06-20-performance-portability-training-series.md:6]: There is a space between '2024-06-20' and 'performance-portability-series.webp', should this be a '-'?

**Review by scottstraughan — CHANGES_REQUESTED** [review]
A few tweaks are needed but looks like a fantastic addition!

File: content/news/2024/2024-06-20-performance-portability-training-series.md (hunk 1 above)
Comment (codeplaymax) [reply content/news/2024/2024-06-20-performance-portability-training-series.md:3]: Coincidentally, it is the correct date!

File: content/news/2024/2024-06-20-performance-portability-training-series.md (hunk 2 above)
Comment (scottstraughan) [reply content/news/2024/2024-06-20-performance-portability-training-series.md:6]: I'll fix this.

**Review by scottstraughan — DISMISSED** [review]

File: content/news/2024/2024-06-20-performance-portability-training-series.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+image: ../../../static/images/news/2024-06-20-performance-portability-series.webp
+tags:
+  - performance
+  - hpc
+  - oneapi
+  - portability
+  - cuda
+---
+
+Portable SYCL code using oneMKL on AMD, Intel, and Nvidia GPUs, presented by CodePlay and Intel, is part of the
```
Comment (carlewis) [inline content/news/2024/2024-06-20-performance-portability-training-series.md:15]:
```suggestion
Portable SYCL code using oneMKL on AMD, Intel, and Nvidia GPUs, presented by Codeplay and Intel, is part of the
```

**Review by carlewis — APPROVED** [review]

**Pull request merged**